### DISCUSSION TITLE
Question about removed submodules

### DISCUSSION THREAD

**Discussion by ulises-jeremias** [cpmech/gosl#55] [Q&A] [answered]
Hi @cpmech ! I just noticed that some useful modules such as ml, graph, plt, etc were removed. Can I ask why did you decided to do that? Is there any plan to reimplement them in the future or maybe move the implementations to a different library? Maybe I can help with any if needed.

**Comment by cpmech** [comment, marked as answer]
Hi There,

These other packages, such as machine learning, plotting, etc., have been removed because they do not depend on CGO and may be developed independently. We can now maintain the core of Gosl more efficiently, which has a focus on the foundation for other scientific code.

The previous version, including more packages, is available here https://github.com/cpmech/gosl/tree/stable-1.1.3 and can be used with the Docker image 1.1.3 as in this hello gosl example https://github.com/cpmech/hello-gosl-old-1.1.3.

I have updated the README file of the main branch with this information (at the bottom).

Thanks again for your interest in Gosl!
Cheers.
Dorival
